campusromero_openedx_extensions/general_custom_views/api/urls.py

```diff
@@ -7,8 +7,9 @@
     url(
         r'^v1/',
         include(
-            'campusromero_openedx_extensions.general_custom_views.api.v1.urls',
-            namespace='camrom-api'
+            ('campusromero_openedx_extensions.general_custom_views.api.v1.urls',
+            'camrom-api'),
+            namespace='camrom-api',
```

Comment:
Why is this needed? 
` include((pattern_list, app_namespace), namespace=None)`
That format receives a pattern list, not the module (correct me if I'm wrong)

Reply:
To be honest, initially I put it like this because I saw it in an example but I was trying to place it in different ways and in all of them I get an error.

Like it was before:
```python
url(
        r'^v1/',
        include(
            'campusromero_openedx_extensions.general_custom_views.api.v1.urls',
            namespace='camrom-api'
        )
```
![Screenshot from 2021-09-17 10-42-46](https://user-images.githubusercontent.com/35668326/133803687-93e4bc34-7bb3-4ffb-97ba-caeb43e005df.png)

Other tests:

```python
url(
        r'^v1/',
        include(
            'campusromero_openedx_extensions.general_custom_views.api.v1.urls',
            app_name='camrom-api',
            namespace='camrom-api'
        )
```
![Screenshot from 2021-09-17 10-44-45](https://user-images.githubusercontent.com/35668326/133803981-a0b58b7b-a6bf-489c-8462-6a1da7545cf6.png)


```python
url(
        r'^v1/',
        include(
            module=campusromero_openedx_extensions.general_custom_views.api.v1.urls,
            namespace='camrom-api'
        )
```
![Screenshot from 2021-09-17 10-47-44](https://user-images.githubusercontent.com/35668326/133804159-56f165ea-5341-4f62-83e0-284ce175999e.png)

```python
url(
        r'^v1/',
        include(
            module='campusromero_openedx_extensions.general_custom_views.api.v1.urls',
            namespace='camrom-api'
        )
```
![Screenshot from 2021-09-17 10-48-36](https://user-images.githubusercontent.com/35668326/133804229-6fcc7f6d-9164-43ad-bc28-d48522d8f962.png)

✅ The one that worked is this option:
```python
url(
        r'^v1/',
        include(
            ('campusromero_openedx_extensions.general_custom_views.api.v1.urls',
            'camrom-api'),
            namespace='camrom-api',
        )
```

Reply:
A module is a list of patterns, as long as it defines the `urlpatterns` list. See https://github.com/django/django/blob/2da029d8540ab0b2e9edcba25c4d46c52853197f/django/urls/conf.py#L35.

If you don't define namespace [it uses app_name](https://github.com/django/django/blob/2da029d8540ab0b2e9edcba25c4d46c52853197f/django/urls/conf.py#L44).